platform/darwin/src/MGLTypes.h

```diff
@@ -107,6 +108,33 @@ NS_INLINE MGLTransition MGLTransitionMake(NSTimeInterval duration, NSTimeInterva
     return transition;
 }
 
+/**
+ A structure containing information about light position.
+ */
+typedef struct MGLPosition {
```

Comment:
Why is the anchor's type called MGLLightAnchor and placed in MGLLight.h but the position's type called MGLPosition and placed in this file? Do we expect the position to be used in other, non-light-related APIs in the future?